src/js/StyledIcon.js

```diff
@@ -45,11 +45,11 @@ const StyledIcon = styled(IconInner)`
   flex: 0 0 auto;
 
   ${({ size = 'medium', theme }) => `
-    width: ${theme.icon.size[size] || size};
-    height: ${theme.icon.size[size] || size};
+    width: ${theme.global.icon.size[size] || size};
```

Comment:
yeah I think this is a breaking change. at least I'm customizing my icons using `theme.icon`. where is this coming from?